src/plots/cartesian/dragbox.js

```diff
@@ -386,6 +385,8 @@ function makeDragBox(gd, plotinfo, x, y, w, h, ns, ew) {
     }
 
     function zoomDone() {
+        updates = {};
```

Comment:
Note that `updates` is also "reset" in 

https://github.com/plotly/plotly.js/blob/7c03fb46d002da6fd85fde6b9cefe23f6d72568e/src/plots/cartesian/dragbox.js#L598-L647

which is called on pan and scroll.